builder.go

```diff
@@ -31,6 +32,21 @@ type Copy struct {
 	// to the executor for handling.
 	Chown string
 	Chmod string
+	// Additional files which need to be created by executor for this
+	// instruction.
+	Files []File
+}
+
+// File defines if any additional file needs to be created
+// by the executor instruction so that specified command
+// can execute/copy the created file inside the build container.
+type File struct {
+	FileDescriptor uint   // If output needs to be redirected to any FileDescriptor
+	Expand         bool   // Needs to expand heredoc `Data` with build args or environment variable
```

Comment:
We don't set this flag, and seem to only provide already-expanded values of `Data`, so I think we can drop this field.